content/scaling.md

```diff
@@ -4,21 +4,45 @@ order: 40
 description: Learn how to improve your app’s performance by scaling
 ---
 
-Scaling and sizing your application is specific to the architecture and performance characteristics of your app. As apps grow they often require more resources to perform well. Galaxy accommodates growth by offering vertical and horizontal resource scaling. Scaling with Galaxy happens without downtime so your users will not experience service interruption.
+Scaling and sizing your application is specific to the architecture and performance characteristics of your app. As apps grow they often require more resources to perform well. Galaxy accommodates growth by offering vertical and horizontal resource scaling.
+
+As long as you maintain one continously running container at all times, scaling with Galaxy happens without downtime, so your users will not experience service interruption.
```

Comment:
"at least one"

but this is sort of confusing, since it might make people not realize that Galaxy *will* do this unless your containers all crash at once